client/src/components/forms/FormTrackEdit/FormTrackEdit.js

```diff
@@ -32,6 +73,7 @@ class FormTrackEdit extends Component {
           autoFocus
           validate={requiredValidate}
         />
+        <FieldArray name="artists" component={this.renderArtists} />
```

Comment:
@jberns FYI - this is what we will use to set up fields that can have multiple values. IIRC the Guests field on shows is this way.